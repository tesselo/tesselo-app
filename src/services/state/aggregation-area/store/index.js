import state from './aggregation-area.state'
import mutations from './aggregation-area.mutations'
import actions from './aggregation-area.actions'
import getters from './aggregation-area.getters'

export default {
  namespaced: true,
  state,
  mutations,
  actions,
  getters
}
