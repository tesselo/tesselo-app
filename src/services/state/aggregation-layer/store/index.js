
import mutations from './aggregation-layer.mutations'
import actions from './aggregation-layer.actions'
import getters from './aggregation-layer.getters'

export default {
  namespaced: true,
  state: {
    total: 0,
    next: null,
    previous: null,
    pageSize: 25,
    rows: [],
    selectedLayer: null
  },
  mutations,
  actions,
  getters
}
