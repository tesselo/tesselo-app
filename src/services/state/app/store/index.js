
import mutations from './app.mutations'
import actions from './app.actions'
import getters from './app.getters'

export default {
  namespaced: true,
  state: {
    isTouch: false,
    isLoading: false
  },
  mutations,
  actions,
  getters
}
