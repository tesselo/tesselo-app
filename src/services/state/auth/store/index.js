import state from './auth.state'
import mutations from './auth.mutations'
import actions from './auth.actions'
import getters from './auth.getters'

export const authMutations = mutations

export default {
  namespaced: true,
  state,
  mutations,
  actions,
  getters
}
