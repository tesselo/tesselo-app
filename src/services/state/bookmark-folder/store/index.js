import state from './bookmark-folder.state'
import mutations from './bookmark-folder.mutations'
import actions from './bookmark-folder.actions'

export default {
  namespaced: true,
  state,
  mutations,
  actions
}
