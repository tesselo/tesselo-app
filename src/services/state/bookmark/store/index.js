import state from './bookmark.state'
import mutations from './bookmark.mutations'
import actions from './bookmark.actions'

export default {
  namespaced: true,
  state,
  mutations,
  actions
}
