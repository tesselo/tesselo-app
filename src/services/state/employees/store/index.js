
import mutations from './employees.mutations'
import actions from './employees.actions'
import getters from './employees.getters'

export default {
  namespaced: true,
  state: {
    employees: [],
    loading: false,
  },
  mutations,
  actions,
  getters
}
