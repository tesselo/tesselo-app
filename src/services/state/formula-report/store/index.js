import state from './formula-report.state'
import mutations from './formula-report.mutations'
import actions from './formula-report.actions'

export default {
  namespaced: true,
  state,
  mutations,
  actions
}
