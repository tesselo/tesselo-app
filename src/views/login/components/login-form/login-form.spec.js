import Vuex from 'vuex'

import LoginForm from './login-form.vue'
import store from '@/services/store'

const localVue = createLocalVue()

localVue.use(Vuex)

describe('Login Form', () => {
  it('should render login form', () => {
    const wrapper = mount(LoginForm, {
      localVue,
      store
    })

    wrapper.vm.login = jest.fn()

    wrapper.setData({
      username: 'test',
      password: 'password'
    })

    wrapper.find('.login-button').trigger('click')

    expect(wrapper.vm.login).toHaveBeenCalled()

    // expect(vm.$el).toMatchSnapshot();
  })
})