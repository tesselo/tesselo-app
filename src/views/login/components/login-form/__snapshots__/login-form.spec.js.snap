// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Form should render login form 1`] = `<!---->`;
